examples/pytorch/sagemaker_official_container/horovod_mnist.py

```diff
@@ -0,0 +1,187 @@
+# Future
```

Comment:
sagemaker_official_container, doesn't sound like good name. zero_code_change was earlier name i guess.

Reply:
Copied over the name from the Tensorflow directory.
I can change it `zero_code_change` and refactor the TF directory structure in a separate PR.